standup/api/views.py

```diff
@@ -0,0 +1,152 @@
+import json
+import logging
+
+from django.http import (
+    HttpResponse,
+    HttpResponseBadRequest,
+    HttpResponseForbidden,
+    HttpResponseServerError,
+)
+
+from django.utils.text import slugify
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic import View
+
+from .models import SystemToken
+from standup.status.models import Project, Status
+from standup.user.models import StandupUser
+
+
+logger = logging.getLogger(__name__)
+
+
+def convert_to_json(resp):
+    """Converts responses into JSON responses"""
+    resp['content_type'] = 'application/json'
+    content = resp.content
+
+    if isinstance(resp, (
+            HttpResponseBadRequest,
+            HttpResponseForbidden,
+            HttpResponseServerError
+    )):
+        # Errors are in the form {'error': 'error string...'}
+        content = {'error': content}
+
+    resp.content = json.dumps(resp.content)
+    return resp
+
+
+class AuthException(Exception):
+    """Authentication exception"""
+    pass
+
+
+class APIView(View):
+    """API view that looks at the world in JSON"""
+    @classmethod
+    def as_view(cls, *args, **kwargs):
+        # API calls use token authentication and don't need csrf bits
+        return csrf_exempt(super(APIView, cls).as_view(*args, **kwargs))
```

Comment:
If you'd rather I think you can just decorate `dispatch` with `csrf_exempt` and not need to override `as_view` at all, since you're overriding `dispatch` below anyway.